### PR TITLE
Update: Fatten Breadcrumb Text

### PR DESCRIPTION
**Pull request by camjc**
- Light weight was hard to read, so lets use medium.
- Make TravisCI run on latest v5 instead of specifically v5.1
- Use active tense in Breadcrumb tests.

### CODE REVIEW

File: test/components/alexandria/BreadcrumbComponentTest.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ describe('BreadcrumbComponent', () => {
     });
   });
 
-  it('should trigger onClick when a node is clicked', () => {
+  it('should trigger onClick when a clicking a node', () => {
```
Comment (pphminions) [inline test/components/alexandria/BreadcrumbComponentTest.js]: `when a clicking a node` extra `a` here I guess. ( I am really not good with grammar though :))

**Comment by pphminions** [comment]
LGTM :cherries:

File: test/components/alexandria/BreadcrumbComponentTest.js (hunk 1 above)
Comment (camjc) [inline test/components/alexandria/BreadcrumbComponentTest.js]: haha. You're totally right. I misread it.

**Comment by omgaz** [comment]
lgtmerging

**Pull request merged**